surface/include/pcl/surface/mls.h

```diff
@@ -52,6 +52,29 @@
 
 namespace pcl
 {
+  /** \brief Data structure used to store the results of the MLS fitting */
+  struct MLSResult
+  {
+    MLSResult () : num_neighbors (), curvature (), valid (false) {}
```

Comment:
My previous comment on this was also to explicitly initialize an curvature as well to make more obvious what's happening
```cpp
MLSResult () : num_neighbors (0), curvature (0.f), valid (false) {}
```